src/components/ComboBox/useListBox.tsx

```diff
@@ -162,54 +168,62 @@ export function useListBox({
         role="listbox"
         aria-hidden={!isExpanded}
       >
-        {options.map((option, i) => {
-          const isActive = activeOptionIndex === i
-          const className = isActive ? 'active' : undefined
-          const { value, label, disabled, isAdding, isSelected } = option
-          if (isAdding) {
-            return (
-              <AddButton
-                key={`add-${label}`}
-                themes={theme}
-                onClick={() => onAdd && onAdd(label)}
-                onMouseOver={() => setActiveOptionIndex(0)}
-                id={addingButtonId}
-                role="option"
-                className={className}
-              >
-                <AddIcon size={14} color={theme.palette.TEXT_LINK} $theme={theme} />
-                <AddText themes={theme}>「{label}」を追加</AddText>
-              </AddButton>
-            )
-          }
-          return (
-            <SelectButton
-              key={`item-${label}`}
-              type="button"
-              themes={theme}
-              disabled={disabled}
-              onClick={() => onSelect({ value, label })}
-              onMouseOver={() => setActiveOptionIndex(i)}
-              id={getOptionId(option)}
-              role="option"
-              className={className}
-              aria-selected={isSelected}
-            >
-              {label}
-            </SelectButton>
-          )
-        })}
+        {isLoading ? (
+          <LoaderWrapper themes={theme}>
+            <Loader />
+          </LoaderWrapper>
+        ) : (
```

Comment:
How about inserting invisible text for notifying "it's loading now..." to screen reader users?

Reply:
I think it is not necessary because the `Loader` component that is used here includes a visually hidden text with "Loading".
https://github.com/kufu/smarthr-ui/blob/master/src/components/Loader/Loader.tsx#L59

How do you think about it?

Reply:
Ahh I got it, so we don't need to insert it here!